perf/CMakeLists.txt

```diff
@@ -122,12 +137,17 @@ create_perf_test(NAME memtx
 create_perf_test_target(TARGET memtx)
 
 add_custom_target(test-c-perf
+                  COMMAND ${CMAKE_CTEST_COMMAND}
+                          ${TARANTOOL_CTEST_FLAGS}
```

Comment:
Does it mean that we running several tests in the parallel (considerring parallel level)?
I suppose this is not what we want for stable and predictable results.
Also, I afraid we should have different timeout values for perf tests and usual tests. So this variable should be set separately for the perfomance tests.
Same for other `TARANTOOL_CTEST_FLAGS` usage.

Reply:
Fixed.

```diff
diff --git a/perf/CMakeLists.txt b/perf/CMakeLists.txt
index 0337f3e05e..18ad0e320c 100644
--- a/perf/CMakeLists.txt
+++ b/perf/CMakeLists.txt
@@ -135,9 +135,14 @@ create_perf_test(NAME memtx
 )
 create_perf_test_target(TARGET memtx)
 
+list(APPEND TARANTOOL_PERF_CTEST_FLAGS
+    --output-on-failure
+    --no-tests=error
+)
+
 add_custom_target(test-c-perf
                   COMMAND ${CMAKE_CTEST_COMMAND}
-                          ${TARANTOOL_CTEST_FLAGS}
+                          ${TARANTOOL_PERF_CTEST_FLAGS}
                           -L ${TEST_SUITE_NAME}
                   DEPENDS ${RUN_PERF_C_TESTS_LIST}
                   COMMENT "Running C performance tests"
@@ -145,7 +150,7 @@ add_custom_target(test-c-perf
 
 add_custom_target(test-perf
                   COMMAND ${CMAKE_CTEST_COMMAND}
-                          ${TARANTOOL_CTEST_FLAGS}
+                          ${TARANTOOL_PERF_CTEST_FLAGS}
                           -L performance
                   COMMENT "Running performance tests"
 )
diff --git a/perf/lua/CMakeLists.txt b/perf/lua/CMakeLists.txt
index 0a707de404..e22263a2c7 100644
--- a/perf/lua/CMakeLists.txt
+++ b/perf/lua/CMakeLists.txt
@@ -86,7 +86,7 @@ create_perf_lua_test(NAME column_insert
 
 add_custom_target(test-lua-perf
                   COMMAND ${CMAKE_CTEST_COMMAND}
-                          ${TARANTOOL_CTEST_FLAGS}
+                          ${TARANTOOL_PERF_CTEST_FLAGS}
                           -L ${TEST_SUITE_NAME}
                   DEPENDS "${RUN_PERF_LUA_TESTS_LIST}"
                   COMMENT "Running Lua performance tests"

```